manifest/armv7l/r/ruby_ruby_libversion.filelist

```diff
@@ -0,0 +1,11 @@
+# Total size: 36269
```

Comment:
yay!

Reply:
Finally fixed this in the ruby buildsystem!